Renaming `options` to `ctx` (#2934)
RobinTail/express-zod-api: README.md

const yourEndpoint = defaultEndpointsFactory
  .addMiddleware(authMiddleware)
  .build({
    handler: async ({ ctx: { user } }) => {
      // user is the one returned by authMiddleware
    }, // ...
  });
```

You can create a new factory by connecting as many middlewares as you want — they will be executed in the specified
order for all the endpoints produced on that factory. You may also use a shorter inline syntax within the
`.addMiddleware()` method, and have access to the output of the previously executed middlewares in chain as `ctx`:

```typescript
import { defaultEndpointsFactory } from "express-zod-api";

const factory = defaultEndpointsFactory
  .addMiddleware(authMiddleware) // add Middleware instance or use shorter syntax:
  .addMiddleware({
    handler: async ({ ctx: { user } }) => ({}), // user from authMiddleware
  });
```

## Context

> In case need to provide your endpoints with a context that does not depend on Request, like non-persistent database

Author: ```suggestion
If you need to provide your endpoints with a context that does not depend on Request, like non-persistent database
```
